<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LightsWidowQt</class>
 <widget class="QDialog" name="LightsWidowQt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Name</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="_lineEdit_name"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="lightview">
            <attribute name="horizontalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="add_pb">
              <property name="text">
               <string> Add Light</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="delete_pb">
              <property name="text">
               <string>Delete Light</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Position</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>X</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Y</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBox_2"/>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Z</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBox_3"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Color</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>R</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QDoubleSpinBox" name="_color_blue"/>
              </item>
              <item row="1" column="3">
               <widget class="QDoubleSpinBox" name="_color_alpha"/>
              </item>
              <item row="0" column="1">
               <widget class="QDoubleSpinBox" name="_color_red"/>
              </item>
              <item row="0" column="3">
               <widget class="QDoubleSpinBox" name="_color_green"/>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>G</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>B</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>A</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="Line" name="line_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QCheckBox" name="ambient_cb">
              <property name="text">
               <string>Ambient</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="1" column="0">
               <widget class="QDoubleSpinBox" name="sb_ambient3"/>
              </item>
              <item row="1" column="1">
               <widget class="QDoubleSpinBox" name="sb_ambient4"/>
              </item>
              <item row="0" column="0">
               <widget class="QDoubleSpinBox" name="sb_ambient1"/>
              </item>
              <item row="0" column="1">
               <widget class="QDoubleSpinBox" name="sb_ambient2"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0" colspan="2">
               <widget class="QCheckBox" name="specular_cb">
                <property name="text">
                 <string>Specular</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QDoubleSpinBox" name="sb_specular3"/>
              </item>
              <item row="1" column="0">
               <widget class="QDoubleSpinBox" name="sb_specular1"/>
              </item>
              <item row="1" column="1">
               <widget class="QDoubleSpinBox" name="sb_specular2"/>
              </item>
              <item row="2" column="1">
               <widget class="QDoubleSpinBox" name="sb_specular4"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="1" column="0">
               <widget class="QDoubleSpinBox" name="sb_diffuse1"/>
              </item>
              <item row="2" column="0">
               <widget class="QDoubleSpinBox" name="sb_diffuse3"/>
              </item>
              <item row="0" column="0" colspan="2">
               <widget class="QCheckBox" name="diffuse_cb">
                <property name="text">
                 <string>Diffuse</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QDoubleSpinBox" name="sb_diffuse2"/>
              </item>
              <item row="2" column="1">
               <widget class="QDoubleSpinBox" name="sb_diffuse4"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
     <zorder>verticalLayoutWidget</zorder>
     <zorder>verticalLayoutWidget</zorder>
     <zorder>horizontalLayoutWidget_4</zorder>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LightsWidowQt</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LightsWidowQt</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
